### PR TITLE
Replace <WuDatepicker> with <AuDatePicker>

### PR DESCRIPTION
**Pull request by Windvis**
This includes the ember-appuniversum update commit from #95 as well to make it easier to test.

### CODE REVIEW

File: app/components/date-time-picker.js
Comment (Windvis) [inline app/components/date-time-picker.js:5]: Hardcoded it like this for now. Should this come from the .yml translation files?

File: app/components/date-time-picker.js
Comment (Windvis) [inline app/components/date-time-picker.js:53]: AuDatePicker / <duet-date-picker> expects the value attribute as ISO-8601 without any time information. Ideally this shouldn't be needed since AuDatePicker can abstract that for us. I think passing the date instances makes more sense.

File: app/components/date-time-picker.js
Comment (Windvis) [inline app/components/date-time-picker.js:61]: Ideally AuDatePicker would simply return a date / undefined / null.

File: app/styles/app.scss
Comment (Windvis) [inline app/styles/app.scss:59]: This conflicted with the styles of the date picker. I think it can just be removed though. Everything seems to be converted to AuDataTable already?

File: app/components/date-time-picker.js
Comment (nvdk) [reply app/components/date-time-picker.js:5]: ah yes, that seems preferable (we use [ember-intl](https://github.com/ember-intl/ember-intl) which has a `t` helper).

File: app/components/date-time-picker.hbs
Comment (Windvis) [inline app/components/date-time-picker.hbs:5]: Another thing I noticed is that typing in dates doesn't seems to work yet, while it does work in the <duet-date-picker> demo. Clearing the date input with backspace also throws an exception in the console since that isn't handled yet by AuDatePicker.

File: app/components/date-time-picker.hbs
Comment (Windvis) [inline app/components/date-time-picker.hbs:3]: I didn't want to add more unrelated code, but I think adding the [`unique-id` helper](https://emberjs.github.io/rfcs/0659-unique-id-helper.html) would be a better solution. There is a [polyfill](https://github.com/ctjhoa/ember-unique-id-helper-polyfill) for it, but [it's very easy to implement](https://github.com/ctjhoa/ember-unique-id-helper-polyfill/blob/master/addon/helpers/unique-id.js) as well so creating it in app is an option as well.

Comment (Windvis) [reply app/components/date-time-picker.hbs:3]: Created a separate PR #90 for this.

Comment (nvdk) [reply app/components/date-time-picker.hbs:3]: I've merged #90 :+1:

File: app/components/date-time-picker.hbs
Comment (nvdk) [reply app/components/date-time-picker.hbs:5]: since this also did not work correctly in the old component, I'm not going to see that as blocking for now

Comment (nvdk) [reply app/components/date-time-picker.hbs:5]: ah doing some more testing, it is a bit worse.

**Comment by Windvis** [comment]
Blocked until I finish the Appuniversum work.

**Comment by Windvis** [comment]
Reworked the PR based on the changes done in [ember-appuniversum](https://github.com/appuniversum/ember-appuniversum/pull/93/files). Once that one is merged / released this PR can be unblocked by bumping the version.

File: app/components/date-time-picker.js
Comment (Windvis) [inline app/components/date-time-picker.js:22]: I didn't include the `placeholder` since that needs to stay in sync with the adapter, and we don't override that one. Ideally the adapter would support the correct formatting based on the locale as well but I don't know how to implement that yet and it's probably not worth the time investment (yet).

File: app/components/date-time-picker.hbs
Comment (Windvis) [reply app/components/date-time-picker.hbs:5]: Should be ok now.

File: app/components/date-time-picker.js
Comment (Windvis) [reply app/components/date-time-picker.js]: Done. I decided to generate the day and month names since that's not something you want to manually configure for each language and the browser has those lists already.

**Review by nvdk — APPROVED** [review]

File: app/styles/app.scss
Comment (nvdk) [reply app/styles/app.scss:59]: @Dietr can you confirm we no longer need these styles? (I didn't see anything of so I assume they are no longer used)

Comment (Dietr) [reply app/styles/app.scss:59]: There was a small issue with the meeting tables but added a fix for that

**Review by Dietr — APPROVED** [review]
Works fine for me. I do see some design issues that we need to fix when using the datepicker in a modal because the new one doesn't render in a wormhole but we can improve on that later.

**Pull request merged**
